lib/vdsm/gluster/cli.py

```diff
@@ -426,10 +426,7 @@ def _parseVolumeInfo(tree):
             value["volumeStatus"] = VolumeStatus.OFFLINE
```

Comment:
This commit and the one before could be squashed, or even better, drop the other one and use this directly with the right solution.

Reply:
Just to explain the reasoning, we don't squash commit when merging the PR, so we try to keep the git history as clean as possible.

In this case I would squash your commits to just 2, one with the solution (your last commit as I'm commenting this), and another with the additional test. Both signed off and properly commented (in the case of the test, a bit of what is being covered by it would help a lot!).

Reply:
Actually, I feel it's better to keep test case and code changes in the same commit so I squashed my commit into 1.